src/world.js

```diff
@@ -21,15 +21,15 @@ export default class LudumWorld {
       .registerSystem(Enemies)
       .registerSystem(GridRenderer);
 
-    for (let y = 0; y < 10; y++) {
-      for (let x = 0; x < 10; x++) {
+    for (let y = 0; y < 6; y++) {
+      for (let x = 0; x < 12; x++) {
```

Comment:
It'd be nice if these arbitrary values that we will end up tuning were centrally located, but game jam.

Reply:
game jam wins